private_sharing/forms.py

```diff
@@ -127,6 +127,13 @@ class Meta:  # noqa: D101
             "deauth_webhook",
         )
 
+    def __init__(self, *args, **kwargs):
```

Comment:
Is this needed? Since this is a ModelForm and this field doesn't allow blank in the database, I thought it was required already.

Reply:
Artifact of the redirect_url change, reverting.